infrastructure/hub-and-spoke-peerings/variables.tf

```diff
@@ -0,0 +1,121 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+variable "hub_project_id" {
+  description = "Hub Project id. Same project can be used for hub and spokes."
+  type        = string
+}
+
+variable "spoke_1_project_id" {
+  description = "Spoke 1 Project id. Same project can be used for hub and spokes."
+  type        = string
+}
+
+variable "spoke_2_project_id" {
+  description = "Spoke 2 Project id. Same project can be used for hub and spokes."
+  type        = string
+}
+
+variable "hub_subnets" {
```

Comment:
I don't like this approach to variables, and much prefer the one in the [vpn](https://github.com/terraform-google-modules/cloud-foundation-fabric/blob/hub-and-spoke-peering/infrastructure/hub-and-spoke-vpns/variables.tf#L41) and shared vpc examples. Are you strongly in favour of keeping this one?